test/playwright/utils/navigation.ts

```diff
@@ -12,6 +12,13 @@ import {
 import { SCREEN_SIZES } from '~/constants/screens'
 
 import enMessages from '~/locales/en.json'
+/*
+ * Ignore this line as ar.json might not be installed on freshly cloned repos.
+ * See issue #1612 for more details.
+ * To download the translation files run: `pnpm i18n`.
+ */
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
 import rtlMessages from '~/locales/ar.json'
```

Comment:
```suggestion

import rtlMessages from '../../locales/ar.json'
```
I can't include line 22 in this suggestion because it wasn't changed. It will need to be deleted.